docs/rfcs/0001-image-watcher.md

```diff
@@ -0,0 +1,91 @@
+- Start Date: 2020-11-18
+- Target Version: 1.0.0
+
+# Summary
+This RFC proposes adding a new feature to trigger Deployment when an image tag stored at a container registry is updated.
+Similar to Flux's [Image Scan](https://docs.fluxcd.io/en/1.19.0/faq/#how-often-does-flux-check-for-new-images).
+
+# Motivation
+The canonical deployment flow with PipeCD is:
+
+1. CI server pushes the generated image to the container registry after app-repo updated.
+2. User updates the config-repo manually.
+
+It is the User's responsibility to automate these steps to be done in a series of actions, while it is quite a bit of painful.
+Therefore, we would like to solve this problem by letting Piped monitor the container registry.
+
+# Detailed design
+Piped periodically compares the image tags defined in git with the latest tags stored in the container registry, and then pushes them to git if there are any deviations.
+
+### Usage
+Note: Where, we call the client that accesses the container registry to `Image Provider`.
```

Comment:
Sorry, what does this mean?

Reply:
Sorry, it means "Where `Image Provider` is the client that accesses the container registry. So I'll change to it.